lib/yoti/activity_details.rb

```diff
@@ -42,5 +42,12 @@ def initialize(receipt, decrypted_profile = nil)
       @user_id = receipt['remember_me_id']
       @outcome = receipt['sharing_outcome']
     end
+
+    public
+        # @return a JSON [Hash] of the address
```

Comment:
Could we change this so it's like the other attributes on this class with the type first, e.g. 
` # @return [Hash] a JSON [Hash] of the address`

Reply:
That looks like a redundant to me.

Reply:
Sorry, I meant `# @return [Hash] a JSON of the address`

Reply:
That makes sense.